nnpackage/schema/circle_schema.fbs

```diff
@@ -257,6 +263,7 @@ table Tensor {
   buffer:uint;
   name:string;  // For debugging and importing back into tensorflow.
   quantization:QuantizationParameters;  // Optional.
+  mx_quantization:MXQuantizationParameters;  // Optional.
```

Comment:
`QuantizationParameters` table has `details` union field to extend quantization parameter type.
You can refer example in TFLite(LiteRT)'s blockwise quantization parameter: https://github.com/tensorflow/tensorflow/commit/6bf1f71afc07cc047b65cce1a686d95c2702bb3f

Reply:
Thanks. I'll take this way. We may not need to update `Tensor` table in the schema. CC @seanshpark